src/containers/TriggerBinding/TriggerBinding.js

```diff
@@ -0,0 +1,321 @@
+    this.fetchData();
+  }
+
+  componentDidUpdate(prevProps) {
+    const { match, webSocketConnected } = this.props;
+    const { namespace, triggerbindingName } = match.params;
+    const {
+      match: prevMatch,
+      webSocketConnected: prevWebSocketConnected
+    } = prevProps;
+    const {
+      namespace: prevNamespace,
+      triggerbindingName: prevTriggerBindingName
+    } = prevMatch.params;
+
+    if (
+      namespace !== prevNamespace ||
+      triggerbindingName !== prevTriggerBindingName ||
+      (webSocketConnected && prevWebSocketConnected === false)
+    ) {
+      this.fetchData();
+    }
+  }
+
+  fetchData() {
+    const { match } = this.props;
+    const { namespace, triggerbindingName } = match.params;
+    this.props.fetchTriggerBinding({ name: triggerbindingName, namespace });
+  }
+
+  handleTabSelect = content => {
+    if (content === 'details') {
+      this.setState({
+        showYAML: false,
+        showDetails: true
+      });
+    } else if (content === 'yaml') {
+      this.setState({
+        showYAML: true,
+        showDetails: false
+      });
+    } else {
+      this.setState({
+        showYAML: false,
+        showDetails: false
+      });
+    }
+  };
+
+  render() {
+    const {
+      intl,
+      error,
+      match,
+      selectedNamespace,
+      triggerBinding
+    } = this.props;
+    const { showYAML, showDetails } = this.state;
+    const { triggerbindingName } = match.params;
+
+    if (error) {
+      return TriggerBindingContainer.notification({
+        kind: 'error',
+        intl
+      });
+    }
+    if (!triggerBinding) {
+      return TriggerBindingContainer.notification({
+        kind: 'error',
+        message: 'TriggerBinding not available',
+        intl
+      });
+    }
+
+    let labels = '';
+    const formattedLabelsToRender = [];
+    if (triggerBinding.metadata.labels) {
+      labels = JSON.stringify(triggerBinding.metadata.labels)
+        .replace('{', '')
+        .replace('}', '')
+        .replace(/['"]+/g, '')
+        .replace('$', '');
+
+      const labelsSplitOnComma = labels.split(',');
+      labelsSplitOnComma.forEach(foo => {
+        const splitOnColon = foo.split(':');
+        formattedLabelsToRender.push(`  ${splitOnColon[0]}: ${splitOnColon[1]}
+ `);
+      });
+    }
+
+    const headersForParameters = [
+      {
+        key: 'name',
+        header: intl.formatMessage({
+          id: 'dashboard.tableHeader.name',
+          defaultMessage: 'Name'
+        })
+      },
+      {
+        key: 'value',
+        header: intl.formatMessage({
+          id: 'dashboard.triggerBinding.value',
+          defaultMessage: 'Value'
+        })
+      }
+    ];
+
+    const rowsForParameters = triggerBinding.spec.params.map(
+      ({ name, value }) => ({
+        id: name,
+        name,
+        value
+      })
+    );
+
+    return (
+      <div id="triggerBinding">
+        <h1>{triggerbindingName}</h1>
+        <Tabs
+          ariaLabel="listbox"
```

Comment:
Carbon already adds the relevant ARIA attributes and roles to the resulting elements, no need for `ariaLabel`, `role`, etc. here.